include/regulator/pi_reg_bt_mesh.h

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Should this file be under at the same place as the Light ctrl as this is a part of that? Who else should use a BT mesh regulator, and if it is not a specific mesh regulator should it be renamed. As this is so BT mesh specific it some what fits better under the bt mesh folder, as it also references to the Light Lightness Control model.

Reply:
The idea is that the regulator is just a completely normal PI regulator, so we should move it in case someone else needs a PI(D) regulator. I named it this way to show that it is implemented from the design in the mesh spec, to remind people that it should keep following the spec. If we don't want to move this implementation, there's not much point in moving anything really.